File: src/main/java/com/scratch/util/GenericResponseModel.java
package com.scratch.util;


import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class GenericResponseModel {

    @JsonProperty("Message")
    private String message = null;
    @JsonProperty("Success")
    private boolean success;


    public GenericResponseModel(boolean success, String message) {
        this.message = message;
        this.success = success;

    }
}

File: src/main/java/com/scratch/util/GenericResponseModel.kt
package com.scratch.util

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
class GenericResponseModel(val success: Boolean = true, val id: String? = null, var records: List<Any>? = null, var errorCode: String? = null, var message: String? = null, var errors: List<Any>? = null)

